plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorTest.java

```diff
@@ -581,6 +581,76 @@ public void testCharTrailingSpace()
         throw new SkipException("Implement test for ClickHouse");
     }
 
+    @Override
+    public void testNativeQuerySimple()
+    {
+        // table function disabled for ClickHouse, because it doesn't provide ResultSetMetaData, so the result relation type cannot be determined
+        assertThatThrownBy(super::testNativeQuerySimple)
+                .hasMessage("line 1:21: Table function system.query not registered");
```

Comment:
Previously it was
```
hasMessage("line 1:21: Table function clickhouse.system.query not registered");
```

Does it mean the message depends on whether the function name is qualified with catalog on the invocation side?

(it probably makes sense, once we have search path)

Reply:
Yes, the message takes the invocation name.